src/pages/lib/graphqlapi/query-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,572 @@ export const meta = {
   title: `Read application data`,
   description: `Learn more about how to read data using Amplify's GraphQL API category`
 };
+
```

Comment:
@renebrandel This one only has the new content under JS and React Native inline filter for the guide. The rest of the platforms use their existing fragments without any guide language or blending.